pkg/reconciler/v1alpha1/taskrun/resources/output_resource_test.go

```diff
@@ -171,7 +171,7 @@ func TestValidOutputResources(t *testing.T) {
 			Name:    "source-copy-source-git-mz4c7",
 			Image:   "override-with-bash-noop:latest",
 			Command: []string{"/ko-app/bash"},
-			Args:    []string{"-args", "cp -r /workspace/source-workspace/. pipeline-task-name"},
+			Args:    []string{"-args", "cp -r /workspace/output/source-workspace/. pipeline-task-name"},
```

Comment:
Not for this patch, but I think all these examples should be changed to use variables interpolation to expose an environment variable to the bash script.
I think hard-coding the `/workspace/....` path in the task definition is not a good practice, and by using it in the examples people will copy paste it.

Reply:
I agree.